unpackaged/vesuvio_calibration/load_calibration_algorithms.py

```diff
@@ -0,0 +1,21 @@
+"""
+  This script imports and subscribes calibration algorithms for the VESUVIO spectrometer.
+
+  Two Mantid algorithms are provided: EVSCalibrationFit and EVSCalibrationAnalysis.
+
+  EVSCalibrationFit is used to fit m peaks to n spectra. The positions of the peaks are esitmated using the supplied instrument parameter
+  file and the d-spacings of the sample (if provided). Support is provided for both Voigt and Gaussian functions.
+
+  EVSCalibrationAnalysis uses EVSCalibrationFit to calculate instrument parameters using the output of the fitting and the and an existing
```

Comment:
There is a typo in "and the and an" and the sentence is a little bit confusing

Reply:
Ammended.